src/views/Patient/MedReqDropDown/etasuStatus/EtasuStatus.tsx

```diff
@@ -41,11 +53,36 @@ const EtasuStatus = (props: EtasuStatusProps) => {
     }
   }, [props.update]);
 
+  const getRequirements = () => {
+    const output = props.remsAdminResponse?.outputParameters?.reference;
+    if (output) {
+      if (output.startsWith('#')) {
+        // contained reference
+        const reference = output.slice(1);
+        if (props.remsAdminResponse?.contained) {
+          const outputParams = props.remsAdminResponse.contained.find(containedResource => {
+            return containedResource.id === reference;
+          }) as EtasuParam;
+          if (outputParams.parameter) {
+            return outputParams.parameter;
+          } else {
+            console.log('unsupported etasu - no parameters found');
+          }
+        }
+      } else {
+        console.log('unsupported etasu - no contained reference');
+      }
+    } else {
+      console.log('unsupported etasu - no output parameter reference');
+    }
+    return []; // do not return undefined
+  };
+
   const status = props.remsAdminResponse?.status;
   let color = '#f7f7f7'; // off-white
-  if (status === 'Approved') {
+  if (status === 'success') {
     color = '#5cb85c'; // green
-  } else if (status === 'Pending') {
+  } else if (status === 'data-required') {
```

Comment:
I don't see any translation between the status values coming back from the GuidanceResponse and what we used to display. Since the status values in the GuidanceResponse were not designed for this use, a mapping was implemented as follows:
'data-required' => 'Pending'
'success' => 'Approved'